corehq/apps/accounting/payment_handlers.py

```diff
@@ -335,28 +353,48 @@ def create_charge(self, amount, card=None, customer=None):
         )
 
     def update_credits(self, payment_record):
-        self.credit_line = CreditLine.add_credit(
-            payment_record.amount, account=self.account, subscription=self.subscription,
-            product_type=self.product_type, feature_type=self.feature_type,
-            payment_record=payment_record,
-        )
+        for feature in self.features:
+            feature_amount = feature['amount']
+            if feature_amount >= 0.5:
+                self.credit_lines.append(CreditLine.add_credit(
+                    feature_amount,
+                    account=self.account,
+                    subscription=self.subscription,
+                    feature_type=feature['type'],
+                    payment_record=payment_record,
+                ))
+            else:
+                logger.error("[BILLING] {account} tried to make a payment for {feature} for less than $0.5."
```

Comment:
Should this send an email since it's prompting to follow up? I'm not sure it's a good idea to rely on the fact that our logs are currently collected through email.